ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -53,6 +53,81 @@ tests:
   container:
     clone: true
     from: golangci-lint
+- as: check-commits-count
+  commands: ci/prow/check-commits-count
+  container:
+    clone: true
+    from: src
+  optional: true
+- as: check-api-changes
+  commands: ci/prow/check-api-changes
+  container:
+    clone: true
+    from: src
+  optional: true
```

Comment:
I would make all jobs mandatory.
We can always override broken jobs on PR using `/override <job name>`.